server/src/main/java/org/elasticsearch/snapshots/SnapshotsService.java

```diff
@@ -1132,6 +1132,7 @@ public void deleteSnapshot(final String repositoryName, final String snapshotNam
      */
     private void deleteSnapshot(final Snapshot snapshot, final ActionListener<Void> listener, final long repositoryStateId,
                                 final boolean immediatePriority) {
+        logger.info("Deleting snapshot [{}]", snapshot);
```

Comment:
perhaps small caps so that it's more similar to the other log messages in this class